cmd/query/main.go

```diff
@@ -163,6 +163,8 @@ func main() {
 		storageFactory.AddFlags,
 		app.AddFlags,
 		metricsReaderFactory.AddFlags,
+		// add tenancy flags here to avoid panic caused by double registration in all-in-one
+		tenancy.AddFlags,
```

Comment:
So is just adding flags sufficient to make query propagate tenant?  I was expecting to see some usage of these options.

Reply:
Just the flags were not exposed. I think @esnible mainly tested the all-in-one which exposed flags correctly. 

Query tenancy handler initialization:
* https://github.com/jaegertracing/jaeger/blob/b92aa3edc5b682ef0afa89463b069630f3c273cc/cmd/query/app/http_handler.go#L148
* https://github.com/jaegertracing/jaeger/blob/c1bb2946e670129314bf88d9730d8ed7566766d4/cmd/query/app/server.go#L123